elasticdl_preprocessing/layers/to_ragged.py

```diff
@@ -0,0 +1,58 @@
+import tensorflow as tf
+
+_COMMA_SEP = ","
+
+
+class ToRagged(tf.keras.layers.Layer):
+    """Converts a `Tensor` or `RaggedTensor` to a `RaggedTensor`,
+    dropping ignore_value cells. If the input's dtype is string, split
+    the string elements to convert the input to `RaggedTensor` firstly.
```

Comment:
What's the TF version requirement for this layer?

Reply:
Above TF 2.0.0 and I have added a note.